smart-contracts/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -85,6 +76,7 @@ pub fn execute_swap_non_vault_funds(
             vault_config.swap_max_slippage.denominator(),
         )?;
 
+        // Add swap messages
         swap_msgs.push(swap_msg(
```

Comment:
I understand that this is not part of your change, so no need to address this now. If you feel motivated to (and it doesn't lead to many other files being changed), then it would be nice to not use `env` in the `swap_msg` function, but just provide the contract address (the only part of env that is really used)